cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowUtil.qll

```diff
@@ -7,6 +7,7 @@ private import semmle.code.cpp.dataflow.internal.FlowVar
 private import semmle.code.cpp.models.interfaces.DataFlow
 private import semmle.code.cpp.controlflow.Guards
 private import semmle.code.cpp.dataflow.internal.AddressFlow
+private import semmle.code.cpp.models.interfaces.PointerWrapper
```

Comment:
This line can go.

Reply:
Well spotted. Fixed in cd310eb9d5bae6e73fea21f1b7e8a062add9c9e3.